llvm/test/CodeGen/RISCV/GlobalISel/instruction-select/cmp32.mir

```diff
@@ -0,0 +1,801 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
```

Comment:
This does not match the naming of the other test files in this directory. And the instruction name is icmp not cmp.  Should be icmp-rv32.mir